tools/functional-tester/etcd-tester/tester.go

```diff
@@ -85,13 +85,22 @@ func (tt *tester) runLoop() {
 				s.Cancel()
 			}
 
-			log.Printf("etcd-tester: [round#%d case#%d] waiting 5s for pending PUTs to be committed across cluster...", i, j)
-			time.Sleep(5 * time.Second)
+			// TODO: sleep shorter or longer per failure case
+			log.Printf("etcd-tester: [round#%d case#%d] waiting 30s for pending PUTs to be committed across cluster...", i, j)
+			time.Sleep(30 * time.Second)
```

Comment:
i would suggest to figure out why test exactly we need to sleep longer and put this into its recovery phase.